libs/ui/src/ui_strings/app_strings.tsx

```diff
@@ -1083,6 +1103,30 @@ export const appStrings = {
     <UiString uiStringKey="titleBmdIdleScreen">Are you still voting?</UiString>
   ),
 
+  titleBmdInvalidBallotNoBallotDetected: () => (
+    <UiString uiStringKey="titleBmdInvalidBallotNoBallotDetected">
+      No Ballot Detected
+    </UiString>
+  ),
+
+  titleBmdInvalidBallotWrongElection: () => (
+    <UiString uiStringKey="titleBmdInvalidBallotWrongElection">
+      Wrong Election
+    </UiString>
+  ),
+
+  titleBmdInvalidBallotWrongPrecinct: () => (
+    <UiString uiStringKey="titleBmdInvalidBallotWrongPrecinct">
+      Wrong Precinct
+    </UiString>
+  ),
+
+  titleBmdInvalidBallotWrongTestMode: () => (
+    <UiString uiStringKey="titleBmdInvalidBallotWrongTestMode">
+      Wrong Test Mode
```

Comment:
"Test Mode" seems oddly opinionated when we use this to refer to both test and live ballot mode, but we don't really have a way to refer to that mode in a neutral way. If we were to change this, maybe "Ballot Mode"? Just "Mode"?

Reply:
Ah, SGTM